5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -69,16 +69,20 @@ Version 4.x described the levels as L1 ("Minimum"), L2 ("Standard"), and L3 ("Ad
 
 Version 5.0 addresses several issues with this approach:
 
-### Improved Entry Level
+### Easier Entry Level
 
-Feedback indicated that the large number of Level 1 requirements (~120), combined with its designation as the "minimum" level, discouraged adoption. Organizations often either abandoned the standard or implemented only a subset of requirements, undermining the sense of progress and achievement.
+Feedback indicated that the large number of Level 1 requirements (~120), combined with its designation as the "minimum" level that is not good enough for most applications, discouraged adoption. This high barrier to entry made it harder to push adoption of the standard within organizations.
 
 ### The Fallacy of Testability
 
 A key factor in selecting controls for Level 1 in version 4.x was their suitability for assessment through "black box" external penetration testing. However, this approach was not fully aligned with the intent of Level 1 as the minimum set of security controls. Some users argued that Level 1 was insufficient for securing applications, while others found it too difficult to test.
 
 Relying on testability as a criterion is both relative and, at times, misleading. The fact that a requirement is testable does not guarantee that it can be tested in an automated or straightforward manner. Moreover, the most easily testable requirements are not always those with the greatest security impact or the simplest to implement.
 
+As such, in version 5.0 we primarilly made this decision based on risk reduction and effort to implement.
```

Comment:
"As such, in version 5.0, we made this decision primarily based on risk reduction and also keeping in mind the effort to implement."?

Reply:
fixed

Reply:
I tried in my proposal here to point out or make the difference, that "the risk reduction" is the main factor, and the effort to implement can be an additional consideration when deciding, to what level it falls.